pyeudiw/trust/model/trust_source.py

```diff
@@ -0,0 +1,208 @@
+from dataclasses import dataclass
```

Comment:
Faccio ad oggi molta fatica a capire cosa sono e quale sia la differenza tra una TrustSourceData e un TrustParameterData. In particolare TrustParameterData non mi sembra essere valorizzato da nessuna parte e non capisco come dovrei realizzare e usare questo layer di astrazione con implementazioni future.

Chiedo cortesemente se @PascalDR e/o @peppelinux mi possono aiutare con esempi su direct trust / federation / x509.

Reply:
La risposta pare essere fornita all'interno del docstring

non sono sicuro che questo livello di dettaglio nella divisione delle classi sia necessario

Reply:
unisco questa PR e ci torniamo sopra insieme